lib/install/management/create.go

```diff
@@ -45,6 +43,15 @@ func (d *Dispatcher) CreateVCH(conf *config.VirtualContainerHostConfigSpec, sett
 
 	var err error
 
+	// resource pool path determined based on DRS setting.  If enabled then
+	// append the appliance name to the path and a pool will be created.
+	// disabled then no resource pools are supported, so use the provided compute path.
```

Comment:
This comment does a good job summarizing what the following code does, but I think it'd be more helpful to explain _why_ this is the desired behavior.

Reply:
How about this -- possibly overly verbose:
```
// Resource Pools are only available in DRS Enabled environments, so  
// the resource pool path will be determined on that setting.
//
// In a DRS disabled environment resource pools aren't available and all
// VMs will reside in the cluster pool.  Attempting to create a pool would 
// result in an error and vic-machine failure.  
//
// Enabled: 
// append the appliance name to the path with the goal of having the
// pool name match the appliance name.
// Disabled:
// only use the compute path which will avoid a pool creation attempt.
```

Reply:
addressed this in #7690